polyaxon_cli/main.py

```diff
@@ -57,7 +63,12 @@ def cli(context, verbose):
     non_check_cmds = [
         'completion', 'config', 'version', 'login', 'logout', 'deploy', 'admin', 'teardown'
     ]
-    if context.invoked_subcommand not in non_check_cmds:
+    context.obj = context.obj or {}
+    context.obj["offline"] = offline
+    if offline:
+        os.environ['POLYAXON_IS_OFFLINE'] = 'true'
```

Comment:
Minor: can we use booleans instead, `'false'` would be evaluated to `True` as well

Reply:
Actually, you are right, and also the config manager can parse bool strings